packages/slate-editor/src/components/Input/Input.tsx

```diff
@@ -110,6 +112,7 @@ export const Input = forwardRef<HTMLInputElement, Props>(
                             setFocused(false);
                             onBlur?.(event);
                         }}
+                        required={required}
```

Comment:
I had to pass the prop here since it wasn't part of `attributes` to allow having validation on a form. I've also removed `required` from `SearchInputPlaceholder` in order to not render the error state if it's empty. Also doesn't seem like it needs to be required since the value is not used as far as I'm aware 🤔

Reply:
One side effect of this is that the validation is immediately triggered, see:

![Screenshot 2023-01-20 at 17 07 25](https://user-images.githubusercontent.com/4209081/213747333-303871f8-ce2b-43a3-a3d3-f8fe6ca894f4.png)

Maybe we can change it a bit so it only renders the validation state if the input becomes dirty? 🤔

Reply:
> One side effect of this is that the validation is immediately triggered, see:

So maybe the input shouldn't be marked as `required`? AFAIK the button will remain disabled, if it's empty. Right?

Reply:
I am not sure it's gonna be enough. According to [this code](https://github.com/prezly/slate/blob/main/packages/slate-editor/src/components/Input/Input.tsx#L132) it disables the button either when the value is invalid or if required but value is empty.

I think the `required && isEmpty` part is redundant, because empty value on a required input will be caught by the validation.

So, either we can keep it like this, add the dirty check or introduce a prop for disabling the button explicitly.

Reply:
I went with the "dirty" check. Whenever `handleChange` is called, it will set the input as dirty and display the validation afterwards. So no immediate validation state when presented with empty input. Also dropped the extra condition. See https://github.com/prezly/slate/pull/372/commits/d90acface9960eea306ba835aa653308778f8ffa.

Let's see how it works and we can tweak it if necessary.